modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json

```diff
@@ -0,0 +1,88 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.2/shapes/"
```

Comment:
Update to {{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.3/shapes/

Reply:
Done